custom/icds_reports/utils/aggregation_helpers/agg_awc_daily.py

```diff
@@ -0,0 +1,208 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import six
+
+from corehq.apps.userreports.util import get_table_name
+from six.moves import map
+
+from custom.icds_reports.utils.aggregation_helpers import BaseICDSAggregationHelper, transform_day_to_month, \
+    month_formatter
+
+
+class AggAwcDailyAggregationHelper(BaseICDSAggregationHelper):
+    aggregate_parent_table = 'agg_awc_daily'
+
+    def __init__(self, month):
+        self.month = transform_day_to_month(month)
+
+    @property
+    def tablename(self):
+        return "{}_{}".format(self.aggregate_parent_table, self.month.strftime("%Y-%m-%d"))
+
+    def drop_table_query(self):
+        return 'DROP TABLE IF EXISTS "{}"'.format(self.tablename)
+
+    def create_table_query(self):
+        return """
+        CREATE TABLE IF NOT EXISTS "{tablename}" (
+            CHECK (date = %(month_string)s),
+            LIKE "{parent_tablename}" INCLUDING DEFAULTS INCLUDING CONSTRAINTS INCLUDING INDEXES
+        ) INHERITS ("{parent_tablename}")
+        """.format(
+            parent_tablename=self.aggregate_parent_table,
+            tablename=self.tablename,
+        ), {
+            "month_string": month_formatter(self.month),
+        }
+
+    def aggregation_query(self):
+
+        columns = (
+            ('state_id',),
+            ('district_id',),
+            ('block_id',),
+            ('supervisor_id',),
+            ('awc_id',),
+            ('aggregation_level',),
+            ('date', '%(start_date)s'),
```

Comment:
This is going to always be the start of the month because it's the same as the month filter. I think that this should be the actual day its being run for. so month should be '2018-10-1', but this date should be `2018-10-31`